app/lambda/lambda_function.py

```diff
@@ -1,28 +1,57 @@
+# import boto3
+# import os
+# import json
+
+# def lambda_handler(event, context):
+#     print("event: {}".format(event))
+#     sts = boto3.client('sts')
+#     transcribeCredentials = sts.assume_role(
+#         RoleArn=os.environ['TRANSCRIBE_ACCESS_ROLEARN'],
+#         RoleSessionName="access_transcribe_role"
+#     )['Credentials']
+#     print(transcribeCredentials)
+#     result = {}
+#     result['accessKeyId'] = transcribeCredentials['AccessKeyId']
+#     result['secretAccessKey'] = transcribeCredentials['SecretAccessKey']
+#     result['sessionToken'] = transcribeCredentials['SessionToken']
+#     result['region'] = os.environ['AWS_REGION']
+#     return {
+#         "isBase64Encoded": False,
+#         "statusCode": 200,
+#         'body': json.dumps(result),
+#         "headers": {
+#             'Content-Type': 'application/json',
+#             'Access-Control-Allow-Origin': '*',
+#             'Access-Control-Allow-Headers': '*',
+#             'Access-Control-Allow-Methods': 'GET,POST,OPTIONS'
+#             }
+#     }
+
 import boto3
 import os
 import json
+from get_credentials.lambda_function import GetCredentialsLambda
+from create_health_care_professional.lambda_function import CreateHealthCareProfessionalLambda
+from create_patient.lambda_function import CreatePatientLambda
+from list_health_care_professionals.lambda_function import ListHealthCareProfessionalsLambda
+from list_patients.lambda_function import ListPatientsLambda
+from create_session.lambda_function import CreateSessionLambda
+from list_sessions.lambda_function import ListSessionsLambda
 
-def lambda_handler(context, event):
-    print("event: {}".format(event))
-    sts = boto3.client('sts')
-    transcribeCredentials = sts.assume_role(
-        RoleArn=os.environ['TRANSCRIBE_ACCESS_ROLEARN'],
-        RoleSessionName="access_transcribe_role"
-    )['Credentials']
-    print(transcribeCredentials)
-    result = {}
-    result['accessKeyId'] = transcribeCredentials['AccessKeyId']
-    result['secretAccessKey'] = transcribeCredentials['SecretAccessKey']
-    result['sessionToken'] = transcribeCredentials['SessionToken']
-    result['region'] = os.environ['AWS_REGION']
-    return {
-        "isBase64Encoded": False,
-        "statusCode": 200,
-        'body': json.dumps(result),
-        "headers": {
-            'Content-Type': 'application/json',
-            'Access-Control-Allow-Origin': '*',
-            'Access-Control-Allow-Headers': '*',
-            'Access-Control-Allow-Methods': 'GET,POST,OPTIONS'
-            }
-    }
+def lambda_handler(event, context):
+    if(event['resource'] == '/getCredentials'):
```

Comment:
I prefer implementing a hash map/dict to emulate a switch case like behavior instead of if-elseif-if tree flow 
There are some performance benefits that can be seen [here](https://stackoverflow.com/questions/17166074/most-efficient-way-of-making-an-if-elif-elif-else-statement-when-the-else-is-don)

Reply:
Done.